packages/rol/example/PDE-OPT/TOOLS/solver_def.hpp

```diff
@@ -66,6 +66,7 @@
 #include "BelosConfigDefs.hpp"
 #include "BelosLinearProblem.hpp"
 #include "BelosSolverFactory.hpp"
+#include "BelosBlockGmresSolMgr.hpp"
```

Comment:
What's this for?

Reply:
This file directly references BlockGmresSolMgr. BelosSolverFactory.hpp no longer includes them all so I had to readd this.